ttl/cache/BUILD.bazel

```diff
@@ -2,34 +2,48 @@ load("@io_bazel_rules_go//go:def.bzl", "go_library", "go_test")
 
 go_library(
     name = "cache",
-    srcs = ["table.go"],
+    srcs = [
+        "base.go",
+        "infoschema.go",
+        "table.go",
+        "ttlstatus.go",
+    ],
     importpath = "github.com/pingcap/tidb/ttl/cache",
     visibility = ["//visibility:public"],
     deps = [
+        "//infoschema",
         "//parser/ast",
         "//parser/model",
         "//parser/mysql",
+        "//sessionctx",
         "//table/tables",
         "//ttl/session",
         "//types",
         "//util/chunk",
+        "//util/logutil",
         "@com_github_pingcap_errors//:errors",
+        "@org_uber_go_zap//:zap",
     ],
 )
 
 go_test(
     name = "cache_test",
     srcs = [
+        "base_test.go",
+        "infoschema_test.go",
         "main_test.go",
         "table_test.go",
+        "ttlstatus_test.go",
     ],
+    embed = [":cache"],
     flaky = True,
```

Comment:
you can add ```race = "on",``` to enable data race test if you want to do.

Reply:
Thanks for the suggestion. Really helpful :+1: :beers: 

For this PR, all tests are running in a single goroutine, so there is no race. The https://github.com/pingcap/tidb/pull/39472 should benefit from it :smile_cat: